app/Http/Controllers/Admin/StatisticsController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Pterodactyl\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use Pterodactyl\Http\Controllers\Controller;
+use Pterodactyl\Models\Allocation;
+use Pterodactyl\Models\Database;
+use Pterodactyl\Models\Egg;
+use Pterodactyl\Models\Node;
+use Pterodactyl\Models\Server;
+use Pterodactyl\Models\User;
+use JavaScript;
+use Illuminate\Support\Facades\DB;
+use Pterodactyl\Services\DaemonKeys\DaemonKeyProviderService;
+
+class StatisticsController extends Controller
+{
+
+    private $keyProviderService;
+
+    function __construct(DaemonKeyProviderService $keyProviderService)
+    {
+        $this->keyProviderService = $keyProviderService;
+    }
+
+    public function index(Request $request)
+    {
+        $servers = Server::all();
+        $nodes = Node::all();
+        $serversCount = count($servers);
+        $nodesCount = Node::count();
```

Comment:
Can you just use the count function like the $servers?

Reply:
Yes actually